py_ocpi/core/dependencies.py

```diff
@@ -36,7 +36,7 @@ def get_endpoints():
 
 def pagination_filters(
     date_from: datetime = Query(default=None),
-    date_to: datetime = Query(default=datetime.now()),
+    date_to: datetime = Query(default=None),
```

Comment:
Why do you change to None?
Maybe someone using this default value

Reply:
For a high-load server with a large number of stations updating their status at least once every minute or two, this default flag can lead to inconsistent responses.
Example: Requests are made to your server to unload all 200,000 stations, while stations simultaneously update their status. Due to this flag, some stations will be omitted from the response because they will update their status during the request processing.